cmd/gossamer/configcmd_test.go

```diff
@@ -180,48 +179,48 @@ func TestSetP2pConfig(t *testing.T) {
 		description string
 		flags       []string
 		values      []interface{}
-		expected    p2p.Config
+		expected    cfg.P2pCfg
 	}{
 		{
 			"config file",
 			[]string{"config"},
 			[]interface{}{tempFile.Name()},
-			cfgClone.P2pCfg,
+			cfgClone.P2p,
 		},
 		{
 			"no bootstrap, no mdns",
 			[]string{"nobootstrap", "nomdns"},
 			[]interface{}{true, true},
-			p2p.Config{
+			cfg.P2pCfg{
 				BootstrapNodes: cfg.DefaultP2PBootstrap,
 				Port:           cfg.DefaultP2PPort,
-				RandSeed:       cfg.DefaultP2PRandSeed,
-				NoBootstrap:    true,
-				NoMdns:         true,
+				//RandSeed:       cfg.DefaultP2PRandSeed,
+				NoBootstrap: true,
+				NoMdns:      true,
 			},
 		},
 		{
 			"bootstrap nodes",
 			[]string{"bootnodes"},
 			[]interface{}{"1234,5678"},
-			p2p.Config{
+			cfg.P2pCfg{
 				BootstrapNodes: []string{"1234", "5678"},
 				Port:           cfg.DefaultP2PPort,
-				RandSeed:       cfg.DefaultP2PRandSeed,
-				NoBootstrap:    false,
-				NoMdns:         false,
+				//RandSeed:       cfg.DefaultP2PRandSeed,
+				NoBootstrap: false,
+				NoMdns:      false,
 			},
 		},
 		{
 			"port",
 			[]string{"p2pport"},
 			[]interface{}{uint(1337)},
-			p2p.Config{
+			cfg.P2pCfg{
 				BootstrapNodes: cfg.DefaultP2PBootstrap,
 				Port:           1337,
-				RandSeed:       cfg.DefaultP2PRandSeed,
-				NoBootstrap:    false,
-				NoMdns:         false,
+				//RandSeed:       cfg.DefaultP2PRandSeed,
```

Comment:
should we get rid of this commented out RandSeed?

Reply:
done!